activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/java/org/activiti/cloud/services/modeling/service/ProjectServiceImpl.java

```diff
@@ -387,6 +397,12 @@ private boolean isModelContent(String modelName,
                 && modelTypeService.isContentXML(modelType));
     }
 
+    public List<ModelValidationError> validateProjectName(Project project) {
+        Optional<ProjectValidator> projectNameValidator = projectValidators.stream()
+            .filter(projectValidator -> projectValidator instanceof ProjectNameValidator).findFirst();
+        return projectNameValidator.get().validate(project, null).collect(Collectors.toList());
```

Comment:
Using get on optional *without* checking if present means that if for whatever reason will be empty, it will blow up the code.

Please add a safety net at least:
```
if (projectNameValidator.isEmpty()) {
          return Collections.emptyList();  
        }
```
Alternatively, refactor the stream operations so it does that